mmv1/templates/terraform/post_import/scc_v2_project_big_query_export_config.go.erb

```diff
@@ -0,0 +1,12 @@
+idParts := strings.Split(d.Id(), "/")
+if len(idParts) != 6 {
+  return nil, fmt.Errorf("unexpected format of ID (%q), expected projects/{{project}}/locations/{{location}}/bigQueryExports/{{big_query_export_id}}", d.Id())
+}
+
```

Comment:
reason for not setting location on import like the others?

Reply:
Removed this file.